astropy/cosmology/parameter/tests/test_parameter.py

```diff
@@ -284,7 +280,7 @@ class Example2(Example1):
             def __init__(self, param=15 * u.m):
                 self.param = param
 
-            @Example1.param.validator
+            @Param.validator
```

Comment:
This is a bit weird - would this not affect `Example1.param` as well?

Reply:
No. ``.validator`` returns a new instance, so this is localized to ``Example2``.

Reply:
OK, good to know. Descriptors/properties with subclassing remains often a bit annoying, how you can't super to the super-class setter, etc.